examples/ping-pong/src/main.rs

```diff
@@ -74,6 +74,7 @@ async fn main() {
         waku_host,
         waku_port,
         None,
+        None,
```

Comment:
could you also add an attempt to read the node addresses env var? and instead of using Option<Vec<&str>>, maybe use a simple type Vec<&str> with an empty vec as fallback?

Reply:
That sounds good but I don't think it's a good idea to pass in arrays as env vars, since formatting is tricky, I'd rather leave it to the Radio developers to directly construct a Vec<&str> if they wish to. But you're right that on the SDK side the value could be a Vec<&str> instead of Option<..> and the Radios can just give `vec![]` to it

Reply:
But ofc Radio devs can pass in that argument however they want (if they want to keep the addresses as an env var to keep it private)

Reply:
ummm I think we should have an example of parsing the string here because it is totally reasonable for radio users to pass in a list of nodes and rather inflexible if we expect each developer to construct a Vec of nodes. Radio operators shouldn't be expected to change any of the radio code. It should be relatively easy to make a SDK helper function that take the env var as a string and then split by an empty space or `,`, something like `addresses.split(',').collect();`